_common.sh

```diff
@@ -174,6 +174,28 @@ function get_tomcat_version {
 	echo "${liferay_tomcat_version}"
 }
 
+function invoke_github_api_get_file {
```

Comment:
@kiwm, please move this code to https://github.com/liferay/liferay-docker/blob/master/release/_github.sh and re-use as much as possible code from there. :wink: